[feat] 학습 모델, parameters, metrics → mlflow 저장

lazychoi opened the pull request:
1. 기존 모델 학습 코드가 있던 train_code 폴더 삭제
2. 학습된 모델이 저장되어 있던 trained_model 폴더 삭제
3. mlflow 폴더 : 모델 학습 후 mlflow에 저장하는 코드. main.py, pre_processing.py, train_mlflow.py
4. data 폴더 : hotel_bookings.csv 파일을 hotel_train.csv와 hotel_test.csv로 나눔. hotel_test.csv는 재학습 및 테스트 용도
5. db 폴더 : hotel_train.csv → postgresql, postgresql → DataFrame 변환 코드

lazychoi committed:
feat : mlflow로 db, mlruns에 모델 적재

model_*.py : 모델별 학습 실행
train_mlflow.py : 학습 실행

Issue : #5

lazychoi committed:
feat : mlflow 수정

main.py : 실행 파일
pre_processing.py : 전처리
train_mlflow.py : 모델 학습. mlflow 저장
hotel_train.csv, hotel_test.csv 로 나눔. hotel_test.csv는 재학습용도
train_csv_to_db.py : hotel_train.csv 파일만 db에 적재

Issue : #5

lazychoi committed:
feat : 필요 없는 파일 삭제

train_code 내의 모든 코드 :  mlflow를 이용하지 않았던 코드 모두 삭제
trained_model 내의 모든 파일 : minio를 사용함에 따라 기존에 저장했던 모델 파일 삭제

Issue : #5

AstroNaut000 reviewed (approved):
설명 주석을 제외한 나머지 주석을 제거하면 더 좋겠습니다.

pastnfast commented on mlflow/train_mlflow.py:
```diff
@@ -0,0 +1,61 @@
+    x_data = ct.fit_transform(x)
+    y_data = y.values.reshape(-1,)
+
+    x_train, x_test, y_train, y_test = train_test_split(x_data, y_data, test_size=0.2, stratify=y_data)
```
'_valid'대신 '_test'가 맞을까요?

lazychoi replied:
test가 맞는 것 같아요.
여기서 test는 새로운 데이터로 모델을 실제로 테스트하는 것이 아닌 모델의 성능을 테스트하는 용도라고 생각하면 될 것 같아요.
많은 책에서 그렇게 사용하고 있으니까요.

pastnfast reviewed (changes requested):
LGTM

lazychoi committed:
feat : pipeline 전처리와 모델fit 합침

train_mlflow.py : mlflow로 저장된 모델에 전처리 과정이 담겨있지 않음.
그래서 전처리와 모델 학습을 하나의 pipeline으로 합쳐 fit

Issues : #5

lazychoi committed:
feat : 파이썬 임시 파일 커밋에서 제외

파이썬 임시 파일 커밋에서 제외

lazychoi committed:
feat : db에서 column_name 가져오는 코드

db에서 column_name 가져오는 코드

Issues : #5
